src/Microsoft.DotNet.RemoteExecutor/src/RemoteExecutor.cs

```diff
@@ -99,6 +94,18 @@ static RemoteExecutor()
         private static bool IsNetCore() =>
             Environment.Version.Major >= 5 || RuntimeInformation.FrameworkDescription.StartsWith(".NET Core", StringComparison.OrdinalIgnoreCase);
 
+        /// <summary>
+        /// A timeout (milliseconds) after which a wait on a remote operation should be considered a failure.
+        /// </summary>
+        public static int FailWaitTimeoutMilliseconds
+        {
+            get
+            {
+                int.TryParse(Environment.GetEnvironmentVariable("DOTNET_REMOTE_EXECUTOR_FAIL_WAIT_TIMEOUT_MULTIPLIER"), out int failWaitTimeoutMultiplier);
```

Comment:
```suggestion
                int.TryParse(Environment.GetEnvironmentVariable("DOTNET_TEST_TIMEOUT_MULTIPLIER"), out int failWaitTimeoutMultiplier);
```
I would make the name of the env. variable more generic.